sql/ddl/indices/when-to-add-indexes.md

```diff
@@ -28,8 +28,7 @@ tags:
   - deep
 
 links:
-
-  - '[SQLMag Dos and Donts of indices](http://www.itprotoday.com/microsoft-sql-server/indexing-dos-and-don-ts){website}'
+  - "[SQLMag Do's and Don'ts of indices](http://www.itprotoday.com/microsoft-sql-server/indexing-dos-and-don-ts){website}"
```

Comment:
From [vocabulary.com](https://www.vocabulary.com/articles/wc/dos-and-donts-or-dos-and-donts/) 😂 :

> Unless your editor wishes otherwise, if you write books, spell it dos and don'ts; and if you write for newspapers, magazines, or the Web, spell it do's and don'ts. If you're writing for yourself, spell it any way you want. Just be consistent.

Reply:
nice 😄